### ISSUE TITLE
Problems installing certain packages

### ISSUE THREAD

**Issue by nielsdebruin** [bower/bower#689]
When I tried use bower to install bootstrap using bower install bootstrap I got the following error:

bower bootstrap#3.0.0-rc1       cached git://github.com/twitter/bootstrap.git#3.0.0-rc1
bower bootstrap#3.0.0-rc1     validate 3.0.0-rc1 against git://github.com/twitter/bootstrap.git#3.0.0-rc1
bower bootstrap#3.0.0-rc1      ECMDERR Failed to execute "git ls-remote --tags --heads git://github.com/twitter/bootstrap.git", exit code of #128

What is have tried so far:
I tried a reinstallation and update with npm
I have tried to remove .bower

**Comment by Shawful** [comment]
@conanchen 

I'm experiencing a similar error, but with bower-angular-touch.  Which file do I modify (as explained in step 2 above) and change "git" to "https"?

I tried commenting out angular-touch from my bower.json file, but then the error existed (after attempting to run <code>grunt build</code>) for angular-messages.

My bower.json file is below:

![image](https://cloud.githubusercontent.com/assets/5322272/7478667/d5cb7c8c-f321-11e4-9a97-e6a484cd568b.png)

Also, I don't have a ~/.bower folder, although I have tried clearing the cache in my project folder and my .npm folder - with no luck.
